akka-javasdk/src/main/java/akka/javasdk/RetrySettings.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (C) 2021-2024 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.javasdk;
+
+import java.time.Duration;
+
+public class RetrySettings {
```

Comment:
Initially, this was an interface, but I think now is a good moment to discuss whether to move this class to akka libs.